### PR TITLE
Integrated ant.d and styled components

### PR DESCRIPTION
**Pull request by Saumya29**

### CODE REVIEW

File: src/config/ant-theme-vars.less [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// Available theme variables can be found in
```
Comment (anshurathee) [inline src/config/ant-theme-vars.less]: Either add code to this file or delete this file.

Comment (anshurathee) [reply src/config/ant-theme-vars.less]: PS: You cannot delete this file as it is read in webpack config.

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -1,35 +1,50 @@
...
+          <Menu.Item key="home">
+            <Icon type="home" />
+            <NavLink to="/">
+              Home
+            </NavLink>
+          </Menu.Item>
+        }
+        {
+          isAuthenticated ?
+            <Menu.Item key="logout">
```
Comment (andrewlinfoot) [inline src/components/Navbar/Navbar.js]: Declare these at their own stateless functional components. can be in the same file if only used here. Will make your code much easier to read. something like:
`
isAuthenticated ? <LogoutMenuItem /> : <SignUpOrLoginMenuItem />
`

**Comment by Saumya29** [comment]
It was closed by mistake. Will issue a new one with the mentioned changes.

**Pull request closed without merging**